### PR TITLE
Added shorthash to git functions

### PR DESCRIPTION
**Pull request by voronoipotato**
### Description

Grabs the shorthash

### CODE REVIEW

File: src/app/Fake.Tools.Git/Information.fs [hunk 1]
```diff
@@ -80,7 +80,14 @@ module Information =
             match Environment.environVarOrNone "BUILD_SOURCEVERSION" with
             | None -> reraise ()
             | Some s -> s
-
+    /// <summary>
+    /// Returns the short SHA1 of the current <c>HEAD</c>
+    /// </summary>
+    ///
+    /// <param name="repositoryDir">The git repository.</param>
+    let getShortSHA1 repositoryDir =
```
Comment (xperiandri) [inline src/app/Fake.Tools.Git/Information.fs:88]: Maybe `getCurrentShortSHA1`
@baronfel what do you vote for?

**Comment by voronoipotato** [comment]
@xperiandri  what do you think :) ?

**Review by xperiandri — APPROVED** [review]

**Comment by voronoipotato** [comment]
@xperiandri Ah I see, I need to run fantomas on it to merge. I'll handle that after work today.

**Comment by xperiandri** [comment]
@voronoipotato could you format your changes? Build fails

**Comment by xperiandri** [comment]
@DanielRowe1 let's merge this and I will release

**Comment by voronoipotato** [comment]
Thanks! We use the shorthash at my work sometimes so this is pretty nice.

**Comment by voronoipotato** [comment]
I'll format it after work today. Thanks!

**Comment by Numpsy** [comment]
fwiw I think Fantomas might just want empty lines adding before and after the new function

**Comment by xperiandri** [comment]
@voronoipotato

**Comment by Numpsy** [comment]
I tried tweaking the formatting in a separate branch over at https://github.com/Numpsy/FAKE/commit/ad0da7ca26d98fc5f07018e490aab0e404419a8c and the formatting step of the CI build worked - https://github.com/Numpsy/FAKE/actions/runs/7504044055 - though that intermitent macOS build failure happened in the integration tests.

Not sure if I'd be able to update this PR though, or if it'd need a new one off my fork.

**Comment by Numpsy** [comment]
@voronoipotato This addition has hopefully been merged in #2770 now

**Comment by xperiandri** [comment]
@voronoipotato you can delete the branch

**Pull request closed without merging**